### PR TITLE
Added the new line for the API link documentation

### PR DESCRIPTION
**Pull request by percygautam**
## Description

Added the new line for API link documentation.

## Checklist

- [x] Does the PR follow [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
      PR format?
- [ ] Has included a sample plot to visually illustrate the changes? (only for plot-related functions)
- [ ] Is the new feature properly documented with an example?
- [ ] Does the PR include new or updated tests to cover the new feature (using [pytest fixture pattern](
      https://docs.pytest.org/en/latest/fixture.html#fixture))?
- [x] Is the code style correct (follows pylint and black guidelines)?
- [ ] Is the change listed in [changelog](https://github.com/arviz-devs/arviz/blob/master/CHANGELOG.md#v0xx-unreleased )?

### CODE REVIEW

**Comment by percygautam** [comment]
@OriolAbril Is this method for new line okay?

File: doc/sphinxext/gallery_generator.py [hunk 1]
```diff
@@ -39,8 +39,8 @@ def execfile(filename, globals=None, locals=None):
 
 .. image:: {img_file}
 
-**Python source code:** :download:`[download source: {fname}]<{fname}>`
-**API documentation:** {api_name} 
+|   **Python source code:** :download:`[download source: {fname}]<{fname}>`
+|   **API documentation:** {api_name}
```
Comment (OriolAbril) [inline doc/sphinxext/gallery_generator.py]: I would use `\n` before `**API documentation**...`

Comment (percygautam) [reply doc/sphinxext/gallery_generator.py]: I tried to use this, but it doesn't seem to be supported in `rsT`.

Comment (OriolAbril) [reply doc/sphinxext/gallery_generator.py]: Oh, I would have thought that given that the file is written by the python script the newline would be directly present in the rst file, good to know. Then leaving a blank line between two links should do the trick

Comment (percygautam) [reply doc/sphinxext/gallery_generator.py]: Yes, that does the trick. :)

**Pull request merged**